### DISCUSSION TITLE
Blocking/Hiding commens of the selected user

### DISCUSSION THREAD

**Discussion by ufian** [umputun/remark42#1510] [Ideas]
All social networks have the ability to block/hide the comments of the selected user without a ban on the platform. Could remark42 have similar functionality?

Some users are abusive and don't follow basic rules of communication. I'd like to have the possibility to against all their comments.

**Comment by umputun** [comment]
moved to discussions

**Comment by umputun** [comment]
Unless I'm missing something, you are describing the existing "hide" functionality

![320fq04scn-20221007-005127](https://user-images.githubusercontent.com/535880/194477141-6a34eed6-8f8d-42dc-a014-7a9b12acbf60.png)

<img width="597" alt="71vz3pb3it-20221007-005206" src="https://user-images.githubusercontent.com/535880/194477220-1889aca9-7222-44e2-b973-500173dffbe2.png">

**Comment by umputun** [comment]
fyi: this is the original ticket https://github.com/umputun/remark42/issues/291
